README.md

```diff
@@ -2,9 +2,147 @@
+drawing board.
+
+Issues used to track unproven APIs will be labeled as `unproven-api`s in the issue tracker and they
+may also include the labels `needs-impl` and `needs-driver` to signal what's required for them to
+move to the set of proven traits.
+
+## Implementations
+
+These are (WIP) implementations of `embedded-hal` for various platforms. Feel free to send a PR
+adding yours to the list!
+
+### Linux
+
+- [`linux-embedded-hal`]. For the Raspberry Pi and other SBC that expose pins with embedded
+  functionality (SPI, I2C, etc.)
+
+[`linux-embedded-hal`]: https://crates.io/crates/linux-embedded-hal
+
+### Nordic
+
+- [`nrf51-hal`]. Check the [`microbit`] crate for examples that can be run on the [micro:bit]
+
+[`nrf51-hal`]: https://github.com/therealprof/nrf51-hal
+[`microbit`]: https://github.com/therealprof/microbit
+[micro:bit]: http://microbit.org/
+
+### NXP
+
+- [`lpc82x-hal`]
+
+[`lpc82x-hal`]: https://github.com/braun-robotics/rust-lpc82x-hal
+
+### ST Microelectronics
+
+- [`stm32f103xx-hal`]. Contains examples that can be run on the [Blue pill] and similar boards.
+
+[`stm32f103xx-hal`]: https://github.com/japaric/stm32f103xx
+[Blue pill]: wiki.stm32duino.com/index.php?title=Blue_Pill
+
+- [`stm32f30x-hal`]. Check the [`f3`] crate for examples that can be run on the STM32F3DISCOVERY.
+
+[`stm32f30x-hal`]: https://crates.io/crates/stm32f30x-hal
+[`f3`]: https://crates.io/crates/f3
+
+### Texas Instruments
+
+- [`tm4c123x-hal`]
+
+[`tm4c123x-hal`]: https://github.com/thejpster/tm4c123x-hal
+
+You may be able to find even more implementations by searching for the [`embedded-hal-impl`] keyword
+an crates.io. If you publish a `embedded-hal` implementation to crates.io please use that keyword to
+let others more easily find your crate!
+
+[`embedded-hal-impl`]: https://crates.io/keywords/embedded-hal-driver
+
+## Drivers
+
+These are (WIP) platform agnostic drivers that can be used with any of the above implementations to
+interface all sort of external devices like sensors and actuators. Feel free to send a PR adding
+yours to the list!
+
+- [`l3gd20`]. Gyroscope
+
+[`l3gd20`]: https://crates.io/crates/l3gd20
+
+- [`lsm303dlhc`]. Accelerometer + compass
+
+[`lsm303dlhc`]: https://crates.io/crates/lsm303dlhc
+
+- [`mag3110`]. Magnetometer
+
+[`mag3110`]: https://github.com/therealprof/mag3110
+
+- [`mfrc522`].` RFID reader / writer
+
+[`mfrc522`]: https://crates.io/crates/mfrc522
+
+- [`motor-driver`]. Motor drivers like the L298N and the TB6612FNG
+
+[`motor-driver`]: https://github.com/japaric/motor-driver
+
+- [`mpu9250`]. Accelerometer + gyroscope + magnetometer IMU
+
+[`mpu9250`]: https://github.com/japaric/mpu9250
+
+- [`si5351`]. Clock generator
+
+[`si5351`]: https://github.com/ilya-epifanov/si5351
+
+- [`si7021`]. Humidity and temperature sensor
+
+[`si7021`]: https://github.com/wose/si7021
+
+You may be able to find even more implementations by searching for the [`embedded-hal-driver`]
```

Comment:
This paragraph is less easy to miss, since there are no sub-sections, but I'd still move it above the list of links.